From 33cc53823024425300dc1664f642fdeba02fe2ba Mon Sep 17 00:00:00 2001
From: sonan <sonan@ami.com>
Date: Mon, 17 Mar 2025 11:35:40 +0530
Subject: [PATCH] Fix For SEL-Clear-logging from phosphor logging

SEL Clear event is moved from phosphor-logging service to
Event logging Disabled discrete sensor.

Signed-off-by: sonan <sonan@ami.com>
---
 log_manager.hpp | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/log_manager.hpp b/log_manager.hpp
index 613d3bc..ee20ac7 100644
--- a/log_manager.hpp
+++ b/log_manager.hpp
@@ -372,12 +372,9 @@ class Manager : public details::ServerObject<DeleteAllIface, CreateIface>
      */
     void deleteAll() override
     {
-        log<level::INFO>("Deleting all log entries");
         auto numbersOfLogs = manager.eraseAll();
-        std::map<std::string, std::string> additionalData;
-        additionalData.emplace("NUM_LOGS", std::to_string(numbersOfLogs));
-        manager.create(LogsCleared::errName, Severity::Informational,
-                       additionalData);
+        log<level::INFO>("Deleting all log entries",
+                         entry("NUM_LOGS=%d", numbersOfLogs));
     }
 
     /** @brief D-Bus method call implementation to create an event log.
-- 
2.43.0

